src/v2/Apps/Gene/Components/GeneMeta.tsx

```diff
@@ -10,21 +10,21 @@ interface GeneMetaProps {
 
 const GeneMeta: React.FC<GeneMetaProps> = ({ gene }) => {
   const title = `${gene.name} | Artsy`
-  const description = gene.meta.description
+
+  const fallbackDescription =
+    "Explore art onArtsy. Browse works by size, price, and medium."
```

Comment:
**minor**: missing a space between `on` and `Artsy` here.

Reply:
Wonder how could I miss it. I'll fix it now